indexer/ai/src/main/java/com/google/cloudsearch/ai/AISkillSet.java

```diff
@@ -0,0 +1,103 @@
+package com.google.cloudsearch.ai;
+
+import com.google.cloudsearch.exceptions.InvalidConfigException;
+import org.apache.log4j.BasicConfigurator;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.log4j.Logger;
+
+/**
+ * Generates a set of skills based on the configuration.
+ */
+public class AISkillSet {
+
+    private List<AISkill> skillSet;
```

Comment:
lets just call it skills

Reply:
I have some local variable with name skills, any major issue with having the name skillSet?

Reply:
its not a set, its a simple list, even for set I would have suggested same variable name.
We don't really need that local variable.

Reply:
Okay, I will push this change with the next set of changes